packages/http-client-python/eng/scripts/ci/regenerate.ts

```diff
@@ -242,6 +245,9 @@ function addOptions(
   const emitterConfigs: EmitterConfig[] = [];
   for (const config of getEmitterOption(spec)) {
     const options: Record<string, string> = { ...config };
+    if (flags.pyodide) {
```

Comment:
nit: it might be better to keep the flags having the same name. I know we might not do this everywhere, but might as well when adding new flags